File: src/main/java/com/abysalto/middleware/source/dummyjson/dto/DummyMeta.java
package com.abysalto.middleware.source.dummyjson.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

@JsonIgnoreProperties(ignoreUnknown = true)
public record DummyMeta(
        String createdAt,
        String updatedAt,
        String barcode,
        String qrCode
) {
}

File: dotnet/src/Middleware.Infrastructure/Upstream/Dto/DummyMeta.cs
namespace Middleware.Infrastructure.Upstream.Dto;

/// <summary>Raw DummyJSON product metadata (timestamps, barcode, QR code).</summary>
internal sealed record DummyMeta(
    string? CreatedAt,
    string? UpdatedAt,
    string? Barcode,
    string? QrCode);
